### ISSUE TITLE
Visual Bugs: TextBox / AutoComplete

### ISSUE THREAD

**Issue by donlent** [ironmansoftware/powershell-universal#2419]
### Version

3.9.2

### Severity

Visual

### Steps to Reproduce

Hi all

Here some more visual anomalies with Textboxes and Autocomplete.
I hope the Screenshots and example code lines are clear enough to reproduce, otherwise please let me know. :-)

![image](https://github.com/ironmansoftware/issues/assets/11975745/80aee134-b8b1-4452-becc-5cc7b69fee3a)

![image](https://github.com/ironmansoftware/issues/assets/11975745/b12882a7-8e21-4e3a-9379-8843dca180a5)

```
New-UDGrid -Container -Content {
                    New-UDGrid -Item -LargeSize 3 -ExtraLargeSize 3 -ExtraSmallSize 12 -Content {
                        # VM Specs
                        $VMSpecsHeader = New-UDCardHeader -Avatar (New-UDIcon -Icon "computer" -Size lg) -Title 'VM Specifications'
                        $VMSpecsBody = New-UDCardBody -Content {

                            New-UDTextbox -Id "tboxComputerName" -Label "Computername" -Icon (New-UDIcon -Icon "tag" -Size lg) -FullWidth -Mask '/^(?![0-9]{1,15}$)[a-zA-Z0-9-]{1,15}$/' #-Placeholder "Computername"
                            New-UDElement -Tag "br"
                            New-UDElement -Tag "br"
                            New-UDAutocomplete  -Label "# CPU Sockets" -Options @("1", "2", "4") -Icon (New-UDIcon -Icon 'microchip' -Size lg)
                            New-UDElement -Tag "br"
                            New-UDElement -Tag "br"
                            New-UDTextbox -Label "# CPU Cores" -Icon (New-UDIcon -Icon "microchip" -Size lg)
                            New-UDElement -Tag "br"
                            New-UDElement -Tag "br"
                            New-UDTextbox -Label "RAM (MB)" -Icon (New-UDIcon -Icon "memory" -Size lg)
                            New-UDElement -Tag "br"
                            New-UDElement -Tag "br"
                            New-UDTextbox -Label "Disk Size (GB)" -Icon (New-UDIcon -Icon "harddrive" -Size lg)
                
                        }

                        New-UDCard -Id "VMSpecs" -Header $VMSpecsHeader -Body $VMSpecsBody
                    }
                }

```

### Expected behavior

```console
Allignment and UI behavior should be consistent and same like
```

### Actual behavior

```console
Allignment and UI behavior is not consistent
```

**Comment by donlent** [comment]
Dear @adamdriscoll 
Any chance that this could make it into 4.0? 🙏
Sorry but it's really "ugly" to design a nice UI like that... 🙃

**Comment by donlent** [comment]
This probably illustrates the issue even better... I just used the outlined variant on both UDTextBox and UDAutoComplete:
![image](https://github.com/ironmansoftware/issues/assets/11975745/07b0cee4-2b36-4f47-ba94-0ccc185fbd7c)

**Comment by donlent** [comment]
Thanks @adamdriscoll 🙏🏻
Will that also be reflected / merged into 4.x?

**Comment by donlent** [comment]
Never mind @adamdriscoll, I've seen that you mentioned the resolved issue also in the 4.0.8 release notes.
THANKS A LOT!! 🙏😊
